scanning/disa-alignment/shared.py

```diff
@@ -1,12 +1,47 @@
-import re
+from collections import namedtuple
```

Comment:
Per https://github.com/RHSecurityCompliance/contest/blob/main/docs/STYLE.md#import-namespaces , this should probably just `import collections`.
```suggestion
import collections
```